pkg/ctl/cmdutils/gitops.go

```diff
@@ -112,27 +112,28 @@ func NewGitOpsConfigLoader(cmd *Cmd) ClusterConfigLoader {
 		flagsIncompatibleWithoutConfigFile: sets.NewString(),
 	}
 
+	validateErrs := errors.New("")
```

Comment:
Using something like https://github.com/hashicorp/go-multierror would be potentially cleaner and more flexible, especially in case of no error, since we can then return `ErrorOrNil()`:
```go
var result *multierror.Error

// ... accumulate errors here

// Return the `error` only if errors were added to the multierror, otherwise
// return nil since there are no errors.
return result.ErrorOrNil()
```

Reply:
Sounds good, I will make that change. Im always hesitant about adding new packages as a new contributor, but I'm not sure where that comes from.

Reply:
We already transitively have it as a dependency anyway, so it is just about adding the latest version to `go.mod` 🙂 
https://github.com/weaveworks/eksctl/blob/286caebb750beac401aac38548a39783e69f7c9e/go.sum#L540-L542